src/logger/loggerTaskEx.c

```diff
@@ -222,7 +222,7 @@ void loggerTaskEx(void *params)
                 if (!g_loggingShouldRun && is_logging) {
                         logging_stopped();
                         const LoggerMessage logStopMsg = getLogStopMessage();
-#if defined(SDCARD_SUPPORT)
+#if SDCARD_SUPPORT
                         queue_logfile_record(&logStopMsg);
 #endif
                         queueTelemetryRecord(&logStopMsg);
```

Comment:
Yeah... there is a ticket out to murder those.  But its a low priority for the moment as delivery is key.